File: validationtestmicroservice/testdevalidation/src/main/java/com/example/testdevalidation/entity/question.java
package com.example.testdevalidation.entity;

import com.fasterxml.jackson.annotation.JsonIgnore;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;

@Entity
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class question {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer idq;
    private String question;
    private String propvrai;

    @ManyToOne
    @JsonIgnore
    testvalidation validation;
    @JsonIgnore
    @OneToOne(mappedBy = "idq")
    reponse reponses;
    private String prop1;
    private String prop2;
    private String prop3;


}

File: markmyflow-frontEnd/src/app/core/models/question.ts
import { empty } from "rxjs";
import { testvalidation } from "./testvalidation";

export class question {

    idq: number;
    question: string ;
    propvrai: string;
    reponses:any;
    prop1:string;
    prop2:string;
    prop3:string;
    validation:testvalidation;

constructor() {
    this.idq = 0;
    this.question = '';
    this.propvrai = '';
    this.prop1='';
    this.prop2='';
    this.prop3='';
    this.validation ;
         
  }
}